binding/go/rhino_test.go

```diff
@@ -19,11 +19,10 @@ import (
 	"os"
 	"path/filepath"
 	"reflect"
-	"runtime"
 	"testing"
 )
 
-var osName = getOS()
+var osName, _ = getLinuxDetails()
```

Comment:
This makes the tests fail on macOS and Windows. I like the idea of removing the duplicate code from the tests, but we'd need a utility function that returns the correct OS and the correct architecture.